Client Development/Client Scripts/Illumio Max Retry Count onChange.js

```diff
@@ -0,0 +1,10 @@
+function onChange(control, oldValue, newValue, isLoading, isTemplate) {
+    if (isLoading || newValue === '') {
+        return;
+    }
+	newValue = newValue.replaceAll(',','');
```

Comment:
Indentation for these `newValue` assignment lines is off in all 3 files.

Reply:
Will fix it.